packages/extension_google_sign_in_as_googleapis_auth/example/android/app/src/androidTest/java/io/flutter/plugins/DartIntegrationTest.java

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface DartIntegrationTest {}
```

Comment:
We're not running this file at all in this repo. Do we actually need native integration tests for this package?

Reply:
This is just a result of doing a cp from the current implementation of the example app of google_sign_in into here. I thought you were looking at adding Android tests for this repo? I can remove these tests, no problem (there's probably the xcode equivalents somewhere under the ios dir?)

Reply:
We can run whatever tests in the repo that we believe are useful, but generally I wouldn't expect a non-plugin package to need to run native tests. Presumably this package is testable by shimming out the plugin it wraps, and doing Dart unit tests?

Reply:
Yes, there are a couple of unit-tests in the root test directory that are enough (the behavior is very simple).

I think the most valuable thing to do here would be to say: this example app passes the same integration tests as google_sign_in, in an attempt to prove they're functionally identical?

(The only change actually is that this example app uses `package:googleapis` and the standard app uses raw http requests with hand-coded authentication headers)